microsoftgraph/microsoft-graph-explorer-v4
# Fix: Return undefined when account type is not established

## Overview

Fixes #1998 


### Demo
![image](https://user-images.githubusercontent.com/18407044/182647914-27009e34-9394-4f3d-8c7a-82b54c0a0dfd.png)

## Testing Instructions

* Check out this branch and load GE locally
* Sign in with a Microsoft account
* Observe fix
